contracts/UFragmentsPolicy.sol

```diff
@@ -244,11 +244,11 @@ contract UFragmentsPolicy is Ownable {
         Ownable.initialize(owner_);
 
         // deviationThreshold = 0.05e18 = 5e16
-        deviationThreshold = 5 * 10**(DECIMALS - 2);
+        deviationThreshold = 25 * 10**(DECIMALS - 3);
 
-        rebaseFunctionGrowth = int256(3 * (10**DECIMALS));
-        rebaseFunctionUpperPercentage = int256(10 * (10**(DECIMALS - 2))); // 0.1
-        rebaseFunctionLowerPercentage = int256((-10) * int256(10**(DECIMALS - 2))); // -0.1
+        rebaseFunctionGrowth = int256(45 * (10**DECIMALS));
```

Comment:
Noting that the current growth on the deployment is 42:
https://etherscan.io/address/0x1b228a749077b8e307c5856ce62ef35d96dca2ea#readProxyContract#F13

Reply:
We decided in the previous PR to keep it here this way and switch it to 45 in the next deployment